blockchain/state_manager/src/vm_circ_supply.rs

```diff
@@ -63,6 +62,15 @@ impl GenesisInfo {
             ..GenesisInfo::default()
         }
     }
+
+    // Allows generation of the current circulating supply
+    pub fn get_supply<DB: BlockStore>(
+        &self,
+        height: ChainEpoch,
+        state_tree: &StateTree<DB>,
+    ) -> Result<TokenAmount, anyhow::Error> {
+        get_circulating_supply(self, height, state_tree)
```

Comment:
`get_circulating_supply` isn't used anywhere else so we can just inline it here. And you can change the `get_supply` name to the slightly more descriptive `get_circulating_supply`.